HW_1/task2.py

```diff
@@ -0,0 +1,18 @@
+'''
+Задача 2: Найдите сумму цифр трехзначного числа.
+
+*Пример:*
+
+123 -> 6 (1 + 2 + 3)
+100 -> 1 (1 + 0 + 0) |'''
+
+
+n = input ('введите трехзначное число' " ")
+n = int (n)
+if n<100 or n>1000:
```

Comment:
Не забывайте расставлять пробелы.
```suggestion
if n < 100 or n > 1000:
```